js/interface.js

```diff
@@ -92,6 +92,11 @@ $('.add-video').on('click', function(e) {
     Fliplet.Widget.toggleCancelButton(true);
     Fliplet.Widget.toggleSaveButton(true);
     file.video.selectFiles = providerData.data.length === 1 ? providerData.data[0] : providerData.data;
+
+    file.video.bundledFile = _.pick(file.video.selectFiles, [
+      'id', 'path', 'url', 'contentType', 'ext', 'updatedAt'
+    ]);
```

Comment:
@squallstar Does the file object found in `providerData.data` not have the right information? Why did we need to create a separate `bundledFile` property when it came from the same source?

Reply:
Doesn't this change create the follow scenario:

1. Widget instance contains a `selectFiles` reference to a media file, as with all instances of this widget
1. Widget instance doesn't contain a `bundledFile` reference because the user has not opened and saved the widget instance in Studio
1. Widget's `widget.json` points to `bundledFile` as a media file reference
1. When the widget instance is copied to another app, the media file and object referenced in `bundledFile` (which doesn't exist) is copied to the new app with the widget instance data updated accordingly.
1. The widget instance in the new app contains a `selectFiles` reference to a media file, which points to a file that may not have access by this app

Reply:
@tonytlwu you're correct in the sense that we shouldn't be using `selectFiles` for this component because we actually need to bundle the asset.

In other words, all references to `selectFiles`should be removed and the interface should account for that.